hd/lang/lex_utf8.txt

```diff
@@ -28,6 +28,10 @@ fr: dictionnaire des %s
 en: compiled on
 fr: compilé le
 
+    commited on
```

Comment:
I don't want to be rude but ... really ? Like 2 lines above ?
Also, don't you remember that you added those lines in the previous PR I worked on ?